classes/Task/Miscellaneous/UpdateConfig.php

```diff
@@ -97,7 +97,14 @@ public function run(): int
                 return ExitCode::FAIL;
             }
 
-            $xmlVersion = $this->getXmlVersion($fullXmlPath);
+            try {
+                $xmlVersion = $this->extractPrestashopVersionFromXml($fullXmlPath);
+            } catch (Exception $exception) {
+                $this->setErrorFlag();
+                $this->logger->info($this->translator->trans('We couldn\'t find a PrestaShop version in the XML file that was uploaded in your local archive. Please try again'));
```

Comment:
Did you mean `error()`?

```suggestion
                $this->logger->error($this->translator->trans('We couldn\'t find a PrestaShop version in the XML file that was uploaded in your local archive. Please try again'));
```

Reply:
All the "errors" linked to the channel archive are reported in info. Either continue on the same path or we change everything

Reply:
this is returned as a "quickInfo"

Reply:
After discussion, we must leave this as-is to have the result displayed on the interface.